starport_template/lib/pages/wallet_details_page.dart

```diff
@@ -88,27 +87,27 @@ class _WalletDetailsPageState extends State<WalletDetailsPage> {
     );
   }
 
-  void _transferPressed(Balance balance) {
-    final denom = Denom(balance.denom.text);
-    _openSendMoneySheet(denom);
-  }
+  // void _transferPressed(Balance balance) {
+  //   final denom = Denom(balance.denom.text);
+  //   _openSendMoneySheet(denom);
+  // }
 
   Future _fetchWalletBalances() async {
     await StarportApp.walletsStore.getBalances(widget.walletInfo.address);
   }
 
-  Future<void> _openSendMoneySheet(Denom denom) async {
-    final result = await showModalBottomSheet(
-      context: context,
-      builder: (context) => SafeArea(
-        child: SendMoneySheet(
-          denom: denom,
-          walletInfo: widget.walletInfo,
-        ),
-      ),
-    );
-    if (result == true) {
-      StarportApp.walletsStore.getBalances(widget.walletInfo.address);
-    }
-  }
+  // Future<void> _openSendMoneySheet(Denom denom) async {
+  //   final result = await showModalBottomSheet(
+  //     context: context,
+  //     builder: (context) => SafeArea(
+  //       child: SendMoneySheet(
```

Comment:
let's add a `TODO` for deleting `SendMoneySheet` aswell

Reply:
Updated